components/security/authorization.rst

```diff
@@ -98,10 +98,12 @@ AuthenticatedVoter
 ~~~~~~~~~~~~~~~~~~
 
 The :class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\AuthenticatedVoter`
-voter supports the attributes ``IS_AUTHENTICATED_FULLY``, ``IS_AUTHENTICATED_REMEMBERED``,
-and ``IS_AUTHENTICATED_ANONYMOUSLY`` and grants access based on the current
-level of authentication, i.e. is the user fully authenticated, or only based
-on a "remember-me" cookie, or even authenticated anonymously?::
+voter supports the attributes ``IS_ANONYMOUS``, ``IS_REMEMBERED``, ``IS_IMPERSONATED``
+and ``IS_AUTHENTICATED`` and ``IS_AUTHENTICATED_FULLY`` and grants access based on the current
+level of authentication, i.e. is the user authenticated or only based
+on a "remember-me" cookie or even only anonymous?
```

Comment:
We don't want to use `.. code-block:: php` explicitly, can we use `... only anonymous?::` here? 

//cc @javiereguiluz

Reply:
Why don't we want that? I think `?::` looks really weird Sphinx syntax (although it is correct)

Reply:
https://github.com/OskarStark/doctor-rst/issues/9 for more insights

Reply:
Hmm, I don't agree with this result of the discussion then. It'll result in [being rendered as anonymous?:](https://symfony.com/doc/current/components/security/authorization#authenticatedvoter), which looks really ugly and incorrect.

Reply:
Let’s allow it when the previous paragraph ends with a question mark?

That would make sense to me